cypress/integration/account_settings/display/channel_display_mode_spec.js

```diff
@@ -17,7 +17,7 @@ describe('Account Settings > Display > Channel Display Mode', () => {
 
         // Post a message to a channel
         cy.visit('/');
-        cy.postMessage('Test for channel display mode {enter}');
+        cy.postMessage('Test for channel display mode');
```

Comment:
nit: update the correct usage of `cy.postMessage` where the text param don't need to add `{enter}` since the command itself is doing it already.